pbj-core/buildSrc/src/main/kotlin/com.hedera.pbj.conventions.gradle.kts

```diff
@@ -39,9 +39,14 @@ javaModuleDependencies {
     moduleNameToGA.put("com.google.protobuf", "com.google.protobuf:protobuf-java")
     moduleNameToGA.put("org.antlr.antlr4.runtime", "org.antlr:antlr4-runtime")
     moduleNameToGA.put("org.mockito.inline", "org.mockito:mockito-inline")
+    // The following line is commented out because it causes the Gradle build to break
+    // moduleNameToGA.put("org.mockito.junit.jupiter", "org.mockito:mockito-junit-jupiter")
 }
 
 java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
```

Comment:
This should not be required with the `languageVersion.set(JavaLanguageVersion.of(17))` below.

Did you see anything behaving differently without this setting?

Reply:
The above control the `javac` behavior wrt to the produced byte code. The toolchain version below controls what JDK is required to be installed. 

Our engineers have moved to Java 21 LTS and so the JDK required by the toolchain will change, but the produced Java byte code should not change.

Reply:
Okay,  I restored this.